dotnet/src/webdriver/Proxy.cs

```diff
@@ -51,15 +47,15 @@ public enum ProxyKind
         /// <summary>
         /// Use proxy automatic detection.
         /// </summary>
-        AutoDetect = 4,
+        AutoDetect,
 
         /// <summary>
-        /// Use the system values for proxy settings (default on Linux).
+        /// Use the system values for proxy settings.
         /// </summary>
         System,
 
         /// <summary>
-        /// No proxy type is specified.
+        /// No proxy type is specified. This must be changed before use
         /// </summary>
         Unspecified
```

Comment:
I think we can easily remove it. Right now user may write something like:
```csharp
var proxy = new Proxy();
// and then it was expected user should specify proxy kind
```

If we make proxy kind as required in constructor, then everything is resolved at compilation level:
```csharp
var proxy = new Proxy(ProxyKind.Direct);
```

I think it can be done in scope of other PR, seems there is additional PR how to change proxy kind.

Reply:
Well, that's definitely a backwards incompatible change. Plus I figured it would make sense to keep the implementation close to what Java is doing, still.

Reply:
That's why we need to look at this deeper later. Looks not bad to me.